# HG changeset patch
# User Rik <rik@octave.org>
# Date 1540605037 25200
#      Fri Oct 26 18:50:37 2018 -0700
# Node ID 29eb60bd2e81dd21514f43a494734bd8a40172ab
# Parent  456523d3cb4a5003cbbf90c444a969a8ab0e896e
savefig.m: Remove seealso link to openfig which does not exist.

* savefig.m: Remove seealso link to openfig which does not exist.

diff --git a/scripts/plot/util/savefig.m b/scripts/plot/util/savefig.m
--- a/scripts/plot/util/savefig.m
+++ b/scripts/plot/util/savefig.m
@@ -28,17 +28,17 @@
 ## If unspecified, @var{h} is set to the current figure.
 ##
 ## If unspecified, @var{filename} is set to @file{"Untitled.fig"}.  If
 ## @var{filename} does not have an extension, a default extension @file{".fig"}
 ## will be added.
 ##
 ## A third input argument will be accepted but ignored, for Matlab compatibility.
 ##
-## @seealso{hgsave, hdl2struct, openfig}
+## @seealso{hgsave, hdl2struct}
 ## @end deftypefn
 
 function savefig (varargin)
 
   ## Default values for input arguments
   h = [];
   filename = "Untitled.fig";
 
